docs/guides/server/keycloak-truststore.adoc

```diff
@@ -2,20 +2,35 @@
 <#import "/templates/kc.adoc" as kc>
 
 <@tmpl.guide
-title="Configuring trusted certificates for outgoing requests"
-summary="How to configure the {project_name} Truststore to communicate with external services through TLS."
+title="Configuring trusted certificates"
+summary="How to configure the {project_name} Truststore to communicate through TLS."
 includedOptions="">
 
-When {project_name} communicates with external services through TLS, it has to validate the remote server’s certificate in order to ensure it is connecting to a trusted server. This is necessary in order to prevent man-in-the-middle attacks. The certificates of these remote server’s or the CA that signed these certificates must be put in a truststore. This truststore is managed by the Keycloak server.
+When {project_name} communicates with external services or has an incoming connection through TLS, it has to validate the remote certificate in order to ensure it is connecting to a trusted server. This is necessary in order to prevent man-in-the-middle attacks.
 
-The truststore is used when connecting securely to identity brokers, LDAP identity providers, when sending emails, and for backchannel communication with client applications. It is also useful
-when you want to change the policy on how host names are verified and trusted by the server.
+The certificates of these clients or servers, or the CA that signed these certificates, must be put in a truststore. This truststore is then configured for use by Keycloak.
```

Comment:
Can mention that this mostly applies to self-signed certificates

Reply:
Should that also mention internal CAs like kubernetes / openshift?

Reply:
Updated to mention both self-signed and internal CAs.